### DISCUSSION TITLE
Reverse commit history

### DISCUSSION THREAD

**Discussion by psaikido** [jesseduffield/lazygit#2870] [Q&A] [answered]
Is there a way to see the commits from the earliest at the top? Can you jump to the bottom or top of the list?

**Comment by stefanhaller** [comment, marked as answer]
Reversing is not possible, but you can jump to the bottom by pressing `>` (you need to do this twice if you have more than 300 commits), and to the top with `<`.

**Reply by psaikido** [reply]
How would I have found that out for myself? I've been looking through the menus but have not discovered it.

**Reply by psaikido** [reply]
Ah, 'scroll to top/bottom' I see it now.

**Comment by stefanos82** [comment]
I find it weird that lazygit cannot do what git does with `git log --reverse` :confused:
